modules/recovery/CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+# Version 0.1.1
```

Comment:
Again, I don't think this is a new version.

Reply:
Updated 👍🏾